README.org

```diff
@@ -14,45 +14,45 @@
 - [[#quick-feature-overview][Quick Feature Overview]]
 - [[#quick-start][Quick Start]]
 - [[#detailed-feature-list][Detailed Feature List]]
-   - [[#projects-and-workspaces][Projects and Workspaces]]
```

Comment:
Try regenerating the TOC with `org-list-indent-offset` set to 1. That *should* fix all those indent changes.